collections/_posts/2022-11-14-DeformableConvolutionalNetworks.md

```diff
@@ -0,0 +1,130 @@
+---
+layout: review
+title: "Deformable Convolutional Networks"
+tags:  Convolution, recepteive field, atrou
+author: "Nolann Lainé"
+cite:
+    authors: "Jifeng Dai, Haozhi Qi, Yuwen Xiong, Yi Li, Guodong Zhang, Han Hu, Yichen Wei"
+    title:   "Deformable Convolutional Networks"
+    venue:   "Microsoft Research Asia"
```

Comment:
The venue is meant to refer to the conference/journal where the paper was published, not the affiliation of the authors. From the GitHub repo, the paper was published at ICCV 2017.

P.S. The PDF does not always make it clear what the venue was, so often times it requires looking up the paper or the authors :wink: 

```suggestion
    venue:   "ICCV 2017"
```

Reply:
I make a note of it, thanks! :)